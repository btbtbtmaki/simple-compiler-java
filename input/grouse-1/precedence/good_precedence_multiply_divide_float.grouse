main {

  print 6.9 * 5.3 / 2.3 nl;
  print 8.2 * 3.8 / 3.1 nl;
  print 6.7 * 8.5 / 2.7 nl;
  print 2.7 * 1.1 / 7.5 nl;
  print 1.6 * 1.5 / 4.1 nl;
  print 5.3 * 6.2 / 7.6 nl;
  print 6.7 * 1.5 / 7.7 nl;
  print 2.8 * 1.1 / 9.1 nl;
  print 5.6 * 3.5 / 3.9 nl;
  print 8.1 * 4.7 / 3.9 nl;
  print 5.7 * 8.3 / 2.1 nl;
  print 3.3 * 8.2 / 1.6 nl;

  print 7.6 / 8.4 * 6.2 nl;
  print 4.5 / 7.2 * 2.5 nl;
  print 4.2 / 4.5 * 1.7 nl;
  print 4.9 / 3.6 * 3.4 nl;
  print 1.8 / 4.9 * 1.0 nl;
  print 4.7 / 9.4 * 7.3 nl;
  print 5.1 / 2.7 * 5.3 nl;
  print 3.9 / 5.1 * 1.2 nl;
  print 9.9 / 2.9 * 8.3 nl;
  print 6.2 / 6.9 * 7.0 nl;
  print 1.0 / 3.6 * 6.3 nl;
  print 9.4 / 7.9 * 8.5 nl;

  print 3.9 * 6.8 / 2.9 * 4.3 nl;
  print 9.5 * 7.5 / 3.0 * 9.4 nl;
  print 2.1 * 1.2 / 8.1 * 5.0 nl;
  print 6.9 * 1.1 / 1.2 * 9.0 nl;
  print 7.3 * 1.0 / 6.0 * 9.7 nl;
  print 8.3 * 8.9 / 5.7 * 1.3 nl;
  print 9.6 * 9.5 / 4.2 * 5.6 nl;
  print 5.2 * 6.1 / 4.1 * 7.8 nl;
  print 6.0 * 9.9 / 8.1 * 9.4 nl;
  print 5.1 * 8.0 / 3.3 * 5.9 nl;
  print 3.8 * 2.5 / 5.8 * 5.8 nl;
  print 2.6 * 3.6 / 6.5 * 9.7 nl;

  print 2.2 / 1.3 * 7.9 / 4.2 nl;
  print 1.6 / 7.2 * 3.1 / 5.1 nl;
  print 8.8 / 5.2 * 9.3 / 9.3 nl;
  print 4.8 / 3.6 * 2.6 / 8.7 nl;
  print 3.4 / 3.4 * 9.2 / 4.9 nl;
  print 6.9 / 2.4 * 9.4 / 1.2 nl;
  print 1.0 / 7.9 * 6.9 / 2.1 nl;
  print 6.7 / 5.2 * 4.5 / 5.2 nl;
  print 9.7 / 1.1 * 1.9 / 9.5 nl;
  print 2.1 / 8.1 * 6.5 / 1.8 nl;
  print 2.6 / 2.5 * 3.1 / 1.4 nl;
  print 9.9 / 1.6 * 7.1 / 6.1 nl;

}

