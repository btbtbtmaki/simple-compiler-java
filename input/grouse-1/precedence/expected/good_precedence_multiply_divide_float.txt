15.9
10.0516
21.0926
0.396
0.585366
4.32368
1.30519
0.338462
5.02564
9.76154
22.5286
16.9125
5.60952
1.5625
1.58667
4.62778
0.367347
3.65
10.0111
0.917647
28.3345
6.28986
1.75
10.1139
39.3228
223.25
1.55556
56.925
11.8017
16.8475
121.6
60.3454
68.9333
72.9455
9.5
13.968
3.18315
0.135076
1.69231
0.398467
1.87755
22.5208
0.415913
1.11501
1.76364
0.936214
2.30286
7.20184
